pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -83,7 +85,7 @@ func getDBImages(extensionState testhelpers.TestOptions) (prev string, clean str
 	if err != nil {
 		panic("unable to get docker image version")
 	}
-	return "timescaledev/promscale-extension:0.1.2-ts2-" + pgVersion, dockerImageName, nil
+	return "timescaledev/promscale-extension:0.3.2-2.6.0-pg" + pgVersion, dockerImageName, nil
```

Comment:
why did we need this change again?

Reply:
There were a couple of timescaledb bug fixes we need that show up in 2.6.0. I don't remember at the moment what they were. Seems like one had to do with reassigning ownership of a hypertable. This is the oldest image that has 2.6.0

Reply:
fixed